source/Ship.h

```diff
@@ -280,7 +280,7 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	// Get the number of jumps this ship can make before running out of fuel.
 	// This depends on how much fuel it has and what sort of hyperdrive it uses.
 	// If followParent is false, this ship will not follow the parent.
-	int JumpsRemaining(bool followParent = true) const;
+	bool JumpsRemaining(bool followParent = true) const;
```

Comment:
The comment for this method is no longer accurate. We're no longer returning the number of jumps remaining, but whether the ship has any remaining jumps. Perhaps rename the method to HasJumpsRemaining.